src/layouts/Base.astro

```diff
@@ -60,6 +69,57 @@ const jsonLDSchema = JSON.stringify(
 		<main>
 			<slot />
 		</main>
+		<footer>
+			<div class="sponsors-container">
+				<h2 id="support">Support Collab Lab!</h2>
+				<p>
+					The Collab Lab is
+					<strong>volunteer-driven and free for developers.</strong>
+					We intentionally and actively work to overcome barriers to opportunities
+					for women and under-represented minorities in tech.
+				</p>
+				<p>
+					We make use of free/
+					<a href="https://www.dictionary.com/browse/freemium">freemium</a>
+					tools (i.e. Airtable, Firebase, GitHub, Google Docs, Netlify, Slack, Notion,
+					and Zoom) as much as possible. We would love to have the ability to use
+					upgraded versions of some of these tools to provide a better experience
+					for our volunteers and developers. To this end, we accept
+					<a href="https://github.com/sponsors/the-collab-lab">
+						sponsorships through GitHub
+					</a>. ❤️
+				</p>
+				<iframe
+					height="35"
+					loading="lazy"
+					src="https://github.com/sponsors/the-collab-lab/button"
+					style="border: 0"
+					title="Sponsor the-collab-lab"
+					width="116"></iframe>
+			</div>
+			We also accept direct donations. Please consider supporting The Collab Lab
+			with a one-time donation.
+			<DonationButtons
+				options={donationOptions}
+				enabled={donationsEnabled}
+				client:visible
+			/>
```

Comment:
🔍 Some Astro magic!

1. `DonationButtons` is a Preact component*, and here we are rendering it with everything else. 
2. The `client:visible` directive passed to this component tells Astro "only load this once the user can see it", which means users who don't scroll this far won't waste resources loading it.

*Why Preact, instead of React? React and Preact have a mostly identical API, but Preact is smaller. We get familiarity for the benefit of our community with a much smaller footprint for our users.